cray/mpir.py

```diff
@@ -27,11 +26,13 @@
 import os
 import ctypes
 
+from cray.echo import echo, LOG_INFO, LOG_WARN, LOG_DEBUG, LOG_RAW
```

Comment:
I see `echo` and `LOG_INFO` and `LOG_WARN`.  I don't see any `LOG_DEBUG` or `LOG_RAW` here.  Are these unused imports (perhaps left over from testing / debugging)?

Reply:
Thanks, updated